.gitignore

```diff
@@ -7,3 +7,8 @@ node_modules/
 .DS_Store
 _sass/
 assets/
+/*.ico
+/*.png
+/*.svg
```

Comment:
The docs say `A leading slash matches the beginning of the pathname. For example, "/*.c" matches "cat-file.c" but not "mozilla-sha1/sha1.c".` so I think this should stick to the root dir.